doc/evaluation.rst

```diff
@@ -0,0 +1,331 @@
+
+* :math:`\hat{H}` is the average of two human scores :math:`\hat{H_i} = \frac{{H_i}+{H2_i}}{2}` when two scores available or :math:`\hat{H_i} = H_i` when only one score is available. 
+
+* :math:`N_1` is the number of responses with only one human score available (:math:`N_1+N_2=N`)
+
+PRMSE is computed using :ref:`rsmtool.prmse_utils.compute_prmse <prmse_api>`
+
+.. note::
+
+	The formula assigns higher weight to discrepancies between system scores and human score when human score is the average of two human scores than when the human score is based on a single score.
+
+
+Fairness
+~~~~~~~~
+
+Fairness of automated scores is an important component of RSMTool evaluations (see `Madnani et al, 2017 <https://www.aclweb.org/anthology/papers/W/W17/W17-1605/>`_).
+
+When defining the experiment, the user has the option of specifying which subgroups should be considered for such evaluations using :ref:`subgroups<subgroups_rsmtool>` field. These subgroups are then used in all fairness evaluations. 
+
+All fairness evaluations are conducted on the evaluation set. The metrics is only computed for either `raw_trim` or `scale_trim` score (see :ref:`score postprocessing<score_postprocessing>` for further detail) depending on the value of :ref:`use_scaled_predictions<use_scaled_predictions_rsmtool>` in RSMTool or :ref:`scale_with<scale_with_eval>` in RSMEval. 
+
+.. _dsm:
+
+Difference between standardized means for subgroups (DSM)
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This is a standard evaluation used for evaluating subgroup differences. The  metrics are available in :ref:`intermediate files<rsmtool_eval_files>` ``eval_by_<SUBGROUP>``.
+
+DSM is computed in the following way:
+
+1. For each group, get the *z*-score for each response, using the :math:`\bar{H}`, :math:`\bar{M}`, :math:`\sigma_H`, and :math:`\sigma_S` for system and human scores for the whole evaluation set:
+
+:math:`z_{H_{i}} = \frac{H_i - \bar{H}}{\sigma_H}`
+
+:math:`z_{M_{i}} = \frac{M_i - \bar{M}}{\sigma_M}`
+
+Where i = response i
+
+2. For each response, calculate the difference between machine and human scores: :math:`z_{M_{i}} - z_{H_{i}}`
+
+3. Calculate the mean of the difference :math:`z_{M_{i}} - z_{H_{i}}` by subgroup of interest. 
+
+DSM is computed using :ref:`rsmtool.utils.difference_of_standardized_means<dsm_api>` with:
+
+ ``population_y_true_observe_mn`` = :math:`\bar{H}` for the whole evaluation set
+
+ ``population_y_pred_mn`` = :math:`\bar{M}` for the whole evaluation set
+
+ ``population_y_true_observed_sd`` = :math:`\sigma_H` for the whole evaluation set
+
+ ``population_y_pred_sd`` = :math:`\sigma_M` for the whole evaluation set
+
+ .. note::
+
+	In RSMTool v.6 and earlier subgroup differences were computed using :ref:`standardized mean difference <SMD>` with ``method`` set to ``williamson``. Since this metrics was very sensitive to score distributions, these are no longer computed by RSMTool 7.0. 
```

Comment:
```suggestion
	In RSMTool v6.x and earlier, subgroup differences were computed using :ref:`standardized mean difference <SMD>` with the ``method`` argument set to ``"williamson"``. Since the differences computed in this manner were very sensitive to score distributions, RSMTool no longer uses this function to compute subgroup differences starting with v7.0.
```